packages/genesys-spark-chart-components/src/components/beta/gux-chart-line/tests/__snapshots__/gux-chart-line.e2e.ts.snap

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
consider using githubs file filter above to hide `.snap` files to make reviewing the manual changes easier.